industrial_moveit_benchmarking/launch/cartesian_valgrind.launch

```diff
@@ -3,7 +3,9 @@
   <arg unless="$(arg profile)" name="launch_prefix" value="" />
   <arg     if="$(arg profile)" name="launch_prefix" value="valgrind --tool=callgrind" />
 
-  <node name="cartesian_benchmarking_node" pkg="industrial_moveit_benchmarking" type="cartesian_benchmarking_node" launch-prefix="$(arg launch_prefix)">
-    <rosparam command="load" file="$(find stomp_test_kr210_moveit_config)/config/clik_planning.yaml" />
+
+  <rosparam command="load" file="$(find industrial_moveit_benchmarking)/resources/config/clik_planning.yaml" />
```

Comment:
These are no longer private params?

Reply:
Right, it seems that at some point a change was made were it no longer looked for the parameters under the node's private namespace.